qiita_pet/support_files/doc/source/dev/plugins.rst

```diff
@@ -0,0 +1,92 @@
+.. _plugins:
+
+.. index :: plugins
+
+Qiita Plugins
+=============
+
+The Qiita plugins allow to extend the functionality of Qiita by adding new data
+formats or new processing pipelines. The list of officially supported plugins
+can be found in the `Qiita Spots <https://github.com/qiita-spots>`__ GitHub
+organization.
+
+Overview of Qiita's plugin system
+---------------------------------
+
+Qiita's plugin system has been designed to increase Qiita's flexibility to
+process -omics datasets, while allowing plugin developers to create plugins
+using the programming language that they are more comfortable with.
+
+In order to achieve this flexibility, Qiita represents the different -omics
+datasets using ``Artifacts`` where an ``Artifact`` is a file or set of files
+that conceptually represents a dataset. Note that Qiita is not ``Artifact`` or
+``Pipeline`` aware, thus it is up to the Qiita Plugins to define what an
+artifact is and which methods can be applied to it. With the goal of reducing
+community development effort, there are two different types of plugins:
+``Qiita Type Plugins`` and ``Qiita Plugins``.
+
+The ``Qiita Type Plugins`` define new artifact types. These plugins must define
+two operations: ``Validate`` and ``Generate HTML summary``.
+
+Given the artifact type, the :doc:`prep information <../tutorials/prepare-information-files>`
+and the user-uploaded files, the ``Validate`` operation decides if the
+user-uploaded files creates a valid artifact of the given type. If the files do
```

Comment:
user-uploaded files creates a valid artifact of the given type.
->
user-uploaded files conform a valid Artifact of the new data type.